fastdtw/fastdtw.py

```diff
@@ -0,0 +1,166 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from __future__ import absolute_import, division
+import numbers
+import numpy as np
+from collections import defaultdict
+
+from six.moves import xrange
```

Comment:
I mostly left it from the original code but it appears to only be there for xrange so I removed it in the latest push 3ad1e24. I also noticed the "examples" section didn't have a header in the docstring so I added it to d4c3f27.